docs/usage.rst

```diff
@@ -191,7 +191,9 @@ You can also configure the class to extract other / additional claims from the t
 Full Example
 ^^^^^^^^^^^^
 
-Everything combined might look like this:
+Everything combined might look like the following. Important note: the KeycloakConfiguration.verify attribute maps to the 
+[KeycloakOpenID](https://github.com/marcospereirampj/python-keycloak/blob/5957607ad07536b94d878c3ce5d403c212b35220/src/keycloak/keycloak_openid.py#L62) verify
+attribute, which must be either the False bool value or the str path to the CA file used to sign the the FastAPI application cert. The default KeycloakConfiguration.verify vaue is False.
```

Comment:
And just to be certain, this is the part I was referring to before. Its actually the Keycloak certificate (web server) that's being checked here.

Reply:
Cool, yeah, what I pass in is the CA file that is used to sign the cert for both our keycloak-configured app and the host kubernetes cluster

Reply:
Sorry I still think there is a misunderstanding. The `verify` attribute is used for the SSL/TLS connection to Keycloak, it is *not* used to validate any OpenID related certificates used by Keycloak.

Also, it allows three values:

- `True` in which case certificate validation is performed using the standard system certificate store
- `False` in which case no certificate validation is performed
-  A string, in which case certificate validation is performed using the CA bundle specified.

This should be highlighted in the docs (despite being `requests` default), but I somehow believe you mix this up with OpenID certificates like the keys you'd see in Keycloaks Realm Management? I could be wrong about this. So no client certificates are involved here, its simply the SSL certificate for your https://keycloak.your-domain.org instance.

Reply:
@waza-ari I updated the wording, please review when you get a chance